RxGesture/RxGesture.xcodeproj/project.pbxproj

```diff
@@ -499,7 +499,7 @@
 				);
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				INFOPLIST_FILE = "RxGesture-iOS-Tests/Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 11.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
This would require all the users upgrade to Xcode 12 to run this example project and since it is still in the beta releases, not all the users would be able to run this example.